pyrakoon/sequence/steps.py

```diff
@@ -234,3 +234,45 @@ def prefix(self):
         :return:
         """
         return self._prefix
+
+class RangeAssertion:
```

Comment:
Add type annotations to these classes ( # type: (param_type1, param_type2) -> return_type)

Reply:
ok.